.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,8 +22,14 @@ If this is a nontrivial change please briefly describe your implementation so it
 
 <!--
 Add any applicable config, commands, screenshots or other resources
-to make it easy for us to verify this works, e.g. an example serverless.yml
-or AWS CLI commands to trigger something.
+to make it easy for us to verify this works. The easier you make it for us
+to review a PR, the faster we can review and merge it.
+
+Examples:
```

Comment:
Just though about a tiny change here. I would prefer to have the name of the task followed by a dash with an explanation to reduce cognitive load. Something like this:

- serverless.yml - Fully functioning serverless.yml to easily deploy changes
- Screenshots - Showing the difference between your output and the master
- AWS CLI commands - To list AWS resources and show that the correct config is in place
- Other - Anything else that comes to mind to help us evaluate

For me this is easier to read as I see directly what is suggested to add to this PR without reading through the whole sentence and extracting the information.